daemon/daemon.go

```diff
@@ -131,16 +105,45 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) bool {
 		return true
 	}
 
-	if r.Method != "GET" {
+	if r.Method == "GET" {
+		// Guest and user access restricted to GET requests
+		if c.GuestOK {
+			return true
+		}
+
+		if isUser && c.UserOK {
+			return true
+		}
+	}
+
+	// Remaining admin checks rely on identifying peer uid
+	if !isUser {
 		return false
 	}
 
-	if isUser && c.UserOK {
+	if uid == 0 {
+		// Superuser does anything.
 		return true
 	}
 
-	if c.GuestOK {
-		return true
+	if c.PolkitOK != "" {
+		var flags polkit.CheckFlags
+		allowHeader := r.Header.Get(client.AllowInteractionHeader)
+		if allowHeader != "" {
+			if allow, err := strconv.ParseBool(allowHeader); err != nil {
+				logger.Noticef("error parsing %s header: %s", client.AllowInteractionHeader, err)
```

Comment:
Not sure we want this logger.Noticef() here, we could just silently ignore if the client sends us silly data to avoid spamming syslog to much.

Reply:
Above, @niemeyer specifically asked for the logging to be added back (although only if the client actually sends the request header: not if it is missing entirely).  I've pushed an update to the test to verify that we're only logging on a garbage value now.

This isn't the only way clients can cause us to write log messages, and it is probably better to know that there is a misbehaving client.

Reply:
I missed the message from Gustavo earlier, thanks for adding it back!